app/assets/v2/css/profile.css

```diff
@@ -306,3 +310,84 @@
 .no-profile-textcolor {
   color: #666666;
 }
+
+#float_avatar_image{
+  z-index: 9999999;
+  margin-top: -140px;
+}  
+
+#float_avatar_image img{
+  height: 186px;
+  width: 186px;
+  border: 3px solid #FFFFFF;
+  box-shadow: 0 -2px 4px 0 rgba(0,0,0,0.50);
+}  
+
+#float_avatar_image.is_my_profile:hover img{
```

Comment:
if u use .float_avatar_image in the other place here you just need .is_my_profile:hover without need to overwrite.